### ISSUE TITLE
Exercise 06 question 10 and 11

### ISSUE THREAD

**Issue by red-solaris** [PsyTeachR/msc-data-skills#15]
You can get different answers depending how you calculate as one individual in users and users2 shares a user_id "16498" but has a different gender and birthdate in each

**Comment by debruine** [comment]
Good catch! Users on the site could update their registration and I must have downloaded users and users2 on different days. I'll just delete that row from one dataset to avoid confusion. I'll leave this open until I have time to fix that.
